realis.go

```diff
@@ -352,6 +392,15 @@ func AddBasicAuth(config *RealisConfig, username string, password string) {
 	httpTrans.SetHeader("Authorization", "Basic "+basicAuth(username, password))
 }
 
+//
+func Secure(config *RealisConfig, insecure bool) {
```

Comment:
Make these both client options:
```
// Using the word set to avoid name collision with Interface
func Secure(insecure bool) ClientOption {
	return func(config *RealisConfig) {
		config.insecure = inescure
	}
}
```

Same for CertPath

Reply:
Sure will change